jme3-android/src/main/java/com/jme3/asset/plugins/AndroidLocator.java

```diff
@@ -91,7 +91,7 @@ public class AndroidAssetInfo extends AssetInfo {
         private InputStream in;
         private final String assetPath;
         private int resourceId;
-
+        private static final String AssetOpenError="Failed to open asset";
```

Comment:
The variable name and spacing on this line doesn't conform to standard java style guidelines.